test/e2e/suite/nodeadm_test.go

```diff
@@ -576,7 +574,7 @@ func newLoggerForTests() e2e.PausableLogger {
 
 func getPodIdentityS3Bucket(ctx context.Context, cluster string, client *s3v2.Client) (string, error) {
 	listBucketsOutput, err := client.ListBuckets(ctx, &s3v2.ListBucketsInput{
-		Prefix: aws.String("ekshybridci-arch-"),
+		Prefix: aws.String(strings.ToLower(addon.PodIdentityS3Bucket)),
```

Comment:
let's avoid having `ToLower` in two places
if you need it in lower case, let's define it that way in the constant
if we need a second constant, that's ok